updater.go

```diff
@@ -36,8 +36,8 @@ type Context interface {
 	BeforeApply(update Update) error
 	AfterApply(update Update) error
 	Restart() error
-	ReportError(err error, options UpdateOptions)
-	ReportAction(action UpdateAction, options UpdateOptions)
+	ReportError(err error, update *Update, options UpdateOptions)
+	ReportAction(action UpdateAction, update *Update, options UpdateOptions)
```

Comment:
Nope, it's to give the reporting access to the update, which includes the RequestID returned by the find update request.